packages/studio-web/src/app/microphone.service.ts

```diff
@@ -1,132 +1,64 @@
-// -*- typescript-indent-level: 2 -*-
 import { EventEmitter, Injectable } from "@angular/core";
 
 @Injectable({
   providedIn: "root",
 })
 export class MicrophoneService {
   private chunks: Array<Blob> = [];
-  protected recorderEnded = new EventEmitter<Blob>();
-  public recorderError = new EventEmitter<ErrorCase>();
-  // tslint:disable-next-line
-  public recorderState = new EventEmitter<RecorderState>();
-  private _recorderState = RecorderState.INITIALIZING;
+  private recorder: MediaRecorder | null = null;
+  private recorderEnded = new EventEmitter<Blob>();
 
-  constructor() {}
-
-  private recorder: MediaRecorder | null;
-
-  private static guc() {
-    return navigator.mediaDevices.getUserMedia({ audio: true });
-  }
-
-  getUserContent() {
-    return MicrophoneService.guc();
-  }
-
-  startRecording() {
-    if (this._recorderState === RecorderState.RECORDING) {
-      this.recorderError.emit(ErrorCase.ALREADY_RECORDING);
-    }
-    if (this._recorderState === RecorderState.PAUSED) {
+  async startRecording() {
+    if (this.recorder !== null && this.recorder.state == "paused") {
```

Comment:
`=== "paused"`?

Reply:
Hi! The states are defined here https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder/state (which is why it didn't make much sense to replicate them in the service ...)

Reply:
Sorry, I finally realized my question was unclear. I was asking about `==` vs `===` and I'm thinking we want three equal signs here.